### ISSUE TITLE
Associated Windows store app does not appear in Microsoft Application Registration Portal

### ISSUE THREAD

**Issue by rjax871** [microsoft/appcenter#275]
**What App Center service does this affect?**
Set up Windows Notification Services

**Describe the bug**
After associating my App with Windows Store, from Visual Studio, the app I want to push for notification services, does not appear.

**To Reproduce**
1. In Visual Studio Solution Explorer, right-click the UWP project, then select Store -> Associate App with the Store...
2. In the wizard, click Next, sign in with your Windows developer account, type a name for your app in Reserve a new app name, then click Reserve.
3. After completing the app registration, select the new app name, click Next, and then click Associate. This adds the required Windows Store registration information to the application manifest.
4. Navigate to the Microsoft Application Registration Portal, sign-in with your Microsoft account
5. Windows Store app you associated in the previous step not visible

**Expected behavior**
 Windows Store app you associated in the previous step should be visible to make a note of the value under Application Secrets and the Package SID, which you will use to configure your App Center Push backend.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser : Chrome

**Comment by jwargo** [comment]
@rjax871 I apologize, but I don't see how this issue relates to Visual Studio App Center - can you clarify for me? All I see is that you're using Visual Studio and WNS - neither of which have anything to do with App Center.

**Comment by jwargo** [comment]
@rjax871 my apologies for misunderstanding this. Another user notified me that the docs needed updated for UWP app registration were incorrect and as I worked through it, I noticed a related issue, but not this one. When I follow the instructions, my app appears in the list - so it feels like this is an issue with your account or your browser.

what I did notice is that when you then follow the instructions on the page to use the 'new' experience, your app doesn't appear in the Azure Portal. I've asked our team to take a look at this issue.
